custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -290,11 +309,25 @@ def config_logging():
                 Optional("predictionThreshold"): Int(),
             }
         ),
-        Optional("trainingModel"): Map({Optional("trainOnProject"): Str()}),
+        Optional(ModelMetadataKeys.TRAINING_MODEL): Map({Optional("trainOnProject"): Str()}),
+        Optional(ModelMetadataKeys.CUSTOM_PREDICTOR): Any(),
     }
 )
 
 
+def validate_config_fields(model_config, *fields):
+    missing_sections = []
+    for f in fields:
+        if f not in model_config:
+            missing_sections.append(f)
+
+    if len(missing_sections):
```

Comment:
change according to PEP-8 (https://www.python.org/dev/peps/pep-0008/)
![image](https://user-images.githubusercontent.com/51160392/106897227-542d2000-66fb-11eb-9d62-1eb1c45bc74b.png)


```suggestion
    if missing_sections:
```